charts/workspacemanager/templates/deployment.yaml

```diff
@@ -87,7 +89,37 @@ spec:
         - name: cloudsql-sa-creds
           mountPath: /secrets/cloudsql
           readOnly: true
+      {{- if .Values.enableProxy }}
+      - name: oidc-proxy
+        image: "{{ .Values.proxyImage.repository }}:{{ .Values.proxyImage.version }}"
+        ports:
+        - containerPort: 443
+        - containerPort: 80
+        volumeMounts:
+        - mountPath: /etc/apache2/sites-available/site.conf
+          name: siteconf
+          subPath: site.conf
+        - mountPath: /etc/ssl/certs/server.crt
```

Comment:
This is a standard pattern that we use in many places. The only issue with it is that it assumes a single TLS cert key in a given location. The problem with this is that /etc/ssl/certs contain more than one cert, and /etc/ssl/private can contain more than one key.

Notes:
1. If there will only be one TLS cert and key in this location for all of this type of use case, this is not a required change.
2. Changing this naming pattern may require a rethink about how we name and provision TLS artifacts. Therefore, assuming that Note 1 is the case, this can be deferred.

Reply:
Note 1 is true, there should only be one TLS cert + key for the server